charts/scalardl/charts/envoy

```diff
@@ -0,0 +1 @@
+../../envoy
```

Comment:
ditto.

Reply:
I apologize for the inconvenience. I'll delete it.